docs/spec/spec.md

```diff
@@ -121,6 +122,31 @@ Subscription's call parameter._
 
 ---
 
+## kind: ChannelSubscriptionSet
+
+### group: eventing.internal.knative.dev/v1alpha1
+
+_A ChannelSubscriptionSet holds an aggregation of resolved subscriptions for a
+Channel._
```

Comment:
A couple unknowns I'd like to see specified:
- What is responsible for creating this object? Subscription controller, channel provisioner, something else?
- When is it created? Channel creation time or subscription creation time?

Reply:
I thought I had a bit saying the ChannelSubscriptionSet should be created by the Provisioner as part of provisioning the channel, but I don't see it. Will add.